### PR TITLE
created setup and added in orb.rb file

### PR DESCRIPTION
**Pull request by oliverripps**
Trello Card: 
https://trello.com/c/Y1ivBoQM/12-orb-create-current-relative-value-api-using-sinatra-ruby

### CODE REVIEW

File: orbs.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require "sinatra"
+require 'sinatra/json'
```
Comment (Sammidysam) [inline orbs.rb:2]: I am okay with either single quotes or double quotes, but I strongly prefer that only one is used throughout the entire project.

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# orb-server-ruby
```
Comment (Sammidysam) [inline README.md:1]: Could we leave the title?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+db.rb
```
Comment (Sammidysam) [inline .gitignore:1]: Would be good to provide a sample file, like our other secret files have in other projects.

File: orbs.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+require "sinatra/reloader" if development?
+require "sequel"
+require_relative "db.rb"
+
+
+DB = Sequel.connect(adapter: 'mysql2', host: HOST, database: DBNAME, user: USERNAME, password: PASSWORD)
+get '/' do
+    DB.fetch("SELECT name,inet_ntoa(ip) as ip,water_uuid, elec_uuid, elec_rvid, water_rvid, relative_value FROM orbs o inner join relative_values rv on o.water_rvid=rv.id").map do |row|
+      content_type :json
+      output = {
```
Comment (Sammidysam) [inline orbs.rb:12]: If we are only selecting these variables with the SQL, why do we need to restate them all in this hash?  Wouldn't `row` only contain these values?

File: orbs.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        :name=>row[:name],
+        :ip=>row[:ip],
+        :water_uuid=>row[:water_uuid],
+        :elec_uuid=>row[:elec_uuid],
+        :elec_rvid=>row[:elec_rvid],
+        :water_rvid=>row[:water_rvid],
+        :relative_value=>row[:relative_value]
+      }
+      output.to_json
```
Comment (Sammidysam) [inline orbs.rb:21]: Considering `output` is only used once here, why declare the variable at all?

File: orbs.rb [hunk 6]
```diff
@@ -0,0 +1,23 @@
+require "sinatra"
+require 'sinatra/json'
+require "sinatra/reloader" if development?
+require "sequel"
+require_relative "db.rb"
+
+
+DB = Sequel.connect(adapter: 'mysql2', host: HOST, database: DBNAME, user: USERNAME, password: PASSWORD)
```
Comment (Sammidysam) [inline orbs.rb:8]: Very extremely nitpicky, but I feel like there should only be one blank line after `require_relative` and also a blank line between the database declaration and the first route.

**Review by Sammidysam — CHANGES_REQUESTED** [review]
I need you to send me your `db.rb` file so that I can test this.  I realize I must have the wrong credentials or something.  That will help me get a better idea of how everything is functioning.

I'd also like you to message me sometime toward setting up a time for me to introduce how we plan to rewrite the interface in React using this new API.  Nice work.

File: orbs.rb (hunk 4 above)
Comment (oliverripps) [reply orbs.rb:12]: I don't really understand what you mean here.

File: orbs.rb (hunk 5 above)
Comment (oliverripps) [reply orbs.rb:21]: I don't really understand what the alternative would be.

File: orbs.rb (hunk 4 above)
Comment (Sammidysam) [reply orbs.rb:12]: What is the difference between the object `output` and the object `row`?  I simply don't know.

File: orbs.rb (hunk 5 above)
Comment (Sammidysam) [reply orbs.rb:21]: The last line in any Ruby method is what is returned.  Do we really need to save `output` into a variable, or can we condense this into one line that may make it easier to read / more efficient?

**Comment by Sammidysam** [comment]
Awesome!  I was able to run this and I do have some feedback on the output.

I don't think JavaScript will be able to parse this as it stands.  It turns out that instead of outputting a JSON list, we are outputting a certain number of lines of JSON objects.  We need to wrap them all in a JSON list.  The easiest way to do this is to just run `.to_json` on the array itself.  I would say, maybe we don't need to run `.to_json` on each individual object too if we do this.  Not sure.

Also, I don't think it is too early to start thinking about routing.  Do you have any ideas of what a sensible name for this data response would be?

The reasoning for me thinking about routes is that we do need to write to this server (POST) eventually too.

**Comment by Sammidysam** [comment]
I am going to merge this since this project is rather abandoned anyway.

**Pull request merged**